src/main/webapp/profile.jsp

```diff
@@ -0,0 +1,247 @@
+    <%@page language="java" contentType="text/html; charset=ISO-8859-1" pageEncoding="ISO-8859-1" %>
+
+      <%@page import="java.sql.*"%>
+      <%@page import="java.sql.Connection"%>
+      <%@page import="java.sql.DriverManager"%>
+      <%@page import="java.sql.ResultSet"%>
+      <%@page import="javax.servlet.http.*" %>
+
+      <!DOCTYPE html>
+        <html lang="en">
+          <head>
+              <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" 
+              integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+              <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" 
+              integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
+              <meta charset="UTF-8">
+              <title>Profile Page</title>
+          </head>
+
+          <body>
+            <nav class="navbar navbar-dark bg-primary">
+              <div class="container">
+                <span class="navbar-brand mb-0 h1"></span>
+                <p class="h3">WELCOME TO YOUR PROFILE</p>
+                <a href="userLogout" class="btn btn-secondary" role="button">Logout</a>
+              </div>
+            </nav>
+
+              <div class="col-lg-8 mx-auto p-3 py-md-5">
+
+             <%
```

Comment:
You should not write JDBC-specific code and this much code in the JSP file. Try to find some other way of doing this.